docs/config/extras.md

```diff
@@ -90,7 +90,12 @@ As of Stencil v3.0.0, support for IE 11, Edge <= 18, and Safari 10 has begun to
 reach end-of-life. While this flag and its supporting functionality is
 currently available, it will be removed in a future version of Stencil.
 
-### experimentalImportInjection
+### experimentalImportInjection (deprecated)
```

Comment:
I think the caution note below this should be enough - would you mind removing ' (deprecated)'? Otherwise, we'd have to set up redirects from the old url (`/docs/next/config-extras#experimentalimportinjection`) to (`/docs/next/config-extras#experimentalimportinjection-deprecated`)

```suggestion
### experimentalImportInjection
```